### DISCUSSION TITLE
Support of Elasticsearch 8.x

### DISCUSSION THREAD

**Discussion by HJK181** [dadoonet/elasticsearch-beyonder#281] [General]
Hi, are there any plans to support Elasticsearch version >= 8.x?

**Comment by dadoonet** [comment]
Hey. 

I'm going to look at this. It requires some changes as we would pull now the the client and deprecate the old REST client.

**Comment by dadoonet** [comment]
Sounds like I forgot to release the version 8.
So let's have some of the features implemented like #327 and #326 and then release a version soonish.

I'd like to be ready to welcome the version 9 of Elasticsearch.

cc @fbaligand

**Reply by fbaligand** [reply]
Indeed! 😉
Tell us when version for ES 8 will be published on Maven Central! ;)

**Comment by dadoonet** [comment]
The release has been made yesterday. Thanks all!
